bench-tps/src/bench.rs

```diff
@@ -616,18 +613,12 @@ pub fn generate_and_fund_keypairs<T: Client>(
         let (_, fee_calculator) = client.get_recent_blockhash().unwrap();
         let extra =
             lamports_per_account - last_keypair_balance + fee_calculator.max_lamports_per_signature;
-        let total = extra * (keypairs.len() as u64);
+        let total = extra * (1 + keypairs.len() as u64);
```

Comment:
What is the +1 from?

Reply:
The +1 is for the source keypair. I know it doesn't get included in do_bench_tps(), but it is considered as `funded` in the fund_keys() loop.